jose/src/alg.rs

```diff
@@ -0,0 +1,90 @@
+// SPDX-FileCopyrightText: 2022 Profian Inc. <opensource@profian.com>
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Do we actually want this header in all the files?
AFAIK, RustCrypto crates do not have this. Instead, authors are specified in the `authors` field of package metadata along "RustCrypto Developers". Also, crates are dual licensed `MIT OR Apache-2.0`.
See [the RSA crate](https://github.com/RustCrypto/RSA/blob/master/Cargo.toml) for reference.

Reply:
My company has a policy for it in every file. But we can probably make an exception.